src/widget/city_building_ghost.c

```diff
@@ -457,7 +457,7 @@ static void draw_desirability_range(const map_tile *tile, building_type type, in
     if (building_is_statue_garden_temple(type) && building_monument_working(BUILDING_GRAND_TEMPLE_VENUS)) {
         int value_bonus = ((desirability_value / 4) > 1) ? (desirability_value / 4) : 1;
         desirability_value += value_bonus;
-        if (!(type >= BUILDING_GRAND_TEMPLE_CERES && type <= BUILDING_GRAND_TEMPLE_VENUS)) {
+        if (!(type >= BUILDING_GRAND_TEMPLE_CERES && type <= BUILDING_PATHEON)) {
```

Comment:
```suggestion
        if (!(type >= BUILDING_GRAND_TEMPLE_CERES && type <= BUILDING_PANTHEON)) {
```
typo fails the build